src/Symfony/Component/Routing/Annotation/Route.php

```diff
@@ -143,4 +144,14 @@ public function getCondition()
     {
         return $this->condition;
     }
+
+    public function setUtf8($utf8)
+    {
+        $this->utf8 = $utf8;
+    }
+
+    public function isUtf8()
```

Comment:
That would complicate the transition: options are validated and you can't add an unsupported key (no forward compat). This means it wouldn't be possible to write routes in a forward/backward compatible manner in Yml/Xml/Annotations.